### PR TITLE
Email account confirmation 

### PR DESCRIPTION
**Pull request by cengique**

### CODE REVIEW

**Comment by cengique** [comment]
@kiyros entering a non-existent email in the "forgot password" field causes "internal server error"

**Comment by cengique** [comment]
@kiyros unverified users have no way of verifying their accounts. There should be a link to re-send the verification email.

File: src/Analysim.Web/Controllers/AccountController.cs [hunk 1]
```diff
@@ -335,8 +335,8 @@ private async Task SendEmailConfirmationEmail([FromForm] AccountRegisterVM user,
 
         /*
          * Type : POST
-         * URL : /api/account/forgotPassword
-         * Description: Create and return new User
+         * URL : /api/account/ForgotPassword
```
Comment (cengique) [inline src/Analysim.Web/Controllers/AccountController.cs:338]: @kiyros why did you change to uppercase? All other endpoints are lowercase.

File: src/Analysim.Web/Controllers/AccountController.cs [hunk 2]
```diff
@@ -372,7 +372,11 @@ public async Task<IActionResult> ForgotPassword([FromForm] ForgotPasswordVM form
             //}
 
             // If we got this far, something failed, redisplay form
```
Comment (cengique) [inline src/Analysim.Web/Controllers/AccountController.cs:374]: this comment should be changed since you're returning Ok

**Review by cengique — COMMENTED** [review]
Also other small issues:
- change password form shows one cleartext box and one password box, they should both be password boxes
- Remove text "forgot password works" from top of page
- After entering new password, it doesn't route back to login page correctly.

File: src/Analysim.Web/Controllers/AccountController.cs (hunk 1 above)
Comment (kiyros) [reply src/Analysim.Web/Controllers/AccountController.cs]: fixed in recent update, I wrote it as uppercase, then changed it to lowercase to match the code convention

File: src/Analysim.Web/ClientApp/src/app/email-confirmation/email-confirmation.component.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<p>email-confirmation successful, You can now login!</p>
```
Comment (cengique) [inline src/Analysim.Web/ClientApp/src/app/email-confirmation/email-confirmation.component.html:1]: should have checked whether it's successful

**Review by cengique — COMMENTED** [review]
All good.
